README.md

```diff
@@ -1 +1,203 @@
-# Crypto360
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./src/Logo/logo.PNG" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
Hello @iambenkis Thank you for the feedback. I request for more clarification on this because the logo displays fine on my end and also on the deployed site.
kindly clarify. Thanks.

Reply:
@iambenkis Sorry for the misunderstanding, I now get it.